Web/starter_project_iconics/pages/blog/singleBlog.tsx

```diff
@@ -0,0 +1,80 @@
+
+// type Blog  = {
+//     title : string,
+
+
+// }
+
+// interface props {
+//     title : Blog.title,
+
+// };
+
+const singleBlog = ({}) => {
+  return (
+    <div className="bg-white">
+        <h2 className="text-center blog-title pt-4">
+            The essential guide to Competitive Programming
+        </h2>
+        <p className="text-center uppercase blog-sub-title">
+            Programming, tech | 6 min Read
+        </p>
+        <div className="flex h-auto w-5/6  justify-center items-center mx-auto p-8">
+            <img src="/blogImg.png" className="mx-auto" alt="blog image"/>
+        </div>
+
+        <div className="mx-auto p-4">
+            <img src="/small-blog-img.png" className="mx-auto p-4"/>
+            <div className="">
+                <p className="mx-auto text-center uppercase font-normal blog-sub-title">chaltu kebede | software engineer</p>
```

Comment:
it would be nice if you did define all this blog data in one blog object and populated the page with that instead of hard coding the strings
this allows you to only change how the blog object would be loaded (perhaps by using the id that is passed in the url) later when you have a way to load actual blogs